specs/tripSpecs/tripSpecs.js

```diff
@@ -1,2 +0,0 @@
-var express = require('express');
```

Comment:
I don't even know why this file is here.  It is not in my branch or local repo at all.